.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Rust
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install ALSA
+        run: sudo apt-get install libasound2-dev
```

Comment:
This looks unnecessarily copied over from `pedals`?

Reply:
I tried without it, but wmidi seems to require it on linux. Maybe alsa provides midi as well?